guides/common/modules/con_facts-and-nic-filtering.adoc

```diff
@@ -0,0 +1,17 @@
+[id="facts-and-nic-filtering_{context}"]
+= Facts and NIC filtering
+
+Facts are host parameters such as total memory, operating system version, or architecture.
```

Comment:
I thought about that, but had trouble coming up with a good suggestion. Perhaps
```suggestion
Facts describe aspects such as total memory, operating system version, or architecture as reported by the host.
```

The last part is IMHO also important since the OS version is also a host property, so it's available twice in Foreman. It's important to understand the flow of information.